internal/pkg/dashboard/components/talosinfo.go

```diff
@@ -93,6 +99,24 @@ func (widget *TalosInfo) updateNodeData(data resourcedata.Data) {
 			nodeData.stage = formatStatus(res.TypedSpec().Stage.String())
 			nodeData.ready = formatStatus(res.TypedSpec().Status.Ready)
 		}
+	case *runtime.SecurityState:
+		if data.Deleted {
+			nodeData.secureBootState = notAvailable
+		} else {
+			nodeData.secureBootState = formatStatus(res.TypedSpec().SecureBoot)
+		}
+	case *runtime.MountStatus:
+		if data.Deleted {
+			nodeData.statePartitionMountStatus = notAvailable
+			nodeData.ephemeralPartitionMountStatus = notAvailable
```

Comment:
shouldn't the switch below (between ephemeral and state partitions) apply to this as well?

Reply:
yup, missed that

Reply:
fixed